src/scss/objects/_form-toggle.scss

```diff
@@ -186,7 +186,11 @@ $formToggle-checked-text            : $green;
         .o-formToggle-text {
             width: 100%;
             display: block;
-            transition: padding 200ms ease-in-out;
+            transition: padding 100ms ease-in-out;
+
+            @include media('>=mid') {
+                transition: padding 200ms ease-in-out;
```

Comment:
Can you not just set `transition-duration: 200ms` here and it should inherit the rest from the previous transition declaration?

Reply:
yeah - totally overlooked that. Just updated 😃